pkg/analyser/analyzer_test.go

```diff
@@ -1270,6 +1270,7 @@ func addSchemaToRes(res *resource.Resource, repo resource.SchemaRepositoryInterf
 func TestAnalysis_MarshalJSON(t *testing.T) {
```

Comment:
Can you also update `TestAnalysis_UnmarshalJSON` to make sure we are able to unmarshal this new field too.
`UnmarshalJSON` Will probably need an update.

Unmarshalin analysis results will not be used only in acceptance test anymore, in future `driftctl fmt` we will use it too.